sensors/imu/lsm9dsxx.c

```diff
@@ -92,9 +209,43 @@ static int lsm9dsxx_start(sensor_info_t *info)
 
 static int lsm9dsxx_alloc(sensor_info_t *info, const char *args)
 {
+	int err = 0, ntries = 10;
+
+	/* sensor context allocation */
+	lsm9dsxx_ctx_t *ctx = malloc(sizeof(lsm9dsxx_ctx_t));
+	if (ctx == NULL) {
+		return -ENOMEM;
+	}
```

Comment:
```suggestion
	lsm9dsxx_ctx_t *ctx;
	int err = 0, ntries = 10;

	/* sensor context allocation */
	ctx = malloc(sizeof(lsm9dsxx_ctx_t));
	if (ctx == NULL) {
		return -ENOMEM;
	}
```

Since in our convention we separate variables declaration from code execution, this would look more readable.